src/NServiceBus.RabbitMQ/Configuration/RabbitMQTransportSettingsExtensions.cs

```diff
@@ -64,5 +64,17 @@ public static TransportExtensions<RabbitMQTransport> TimeToWaitBeforeTriggeringC
             transportExtensions.GetSettings().Set(SettingsKeys.TimeToWaitBeforeTriggeringCircuitBreaker, waitTime);
             return transportExtensions;
         }
+
+        /// <summary>
+        /// Overrides the default usePublisherCofirms or value set in connection string.
+        /// </summary>
+        /// <param name="transportExtensions"></param>
+        /// <param name="publisherConfirms">Controls if publisher confirms should be used.</param>
+        /// <returns></returns>
+        public static TransportExtensions<RabbitMQTransport> SetPublisherConfirms(this TransportExtensions<RabbitMQTransport> transportExtensions, bool publisherConfirms)
```

Comment:
I'm thinking the name of this method should be `UsePublisherConfirms` instead of `SetPublisherConfirms`.